core/src/test/java/kafka/test/ClusterInstance.java

```diff
@@ -186,6 +187,12 @@ default Set<GroupProtocol> supportedGroupProtocols() {
     default void waitTopicDeletion(String topic) throws InterruptedException {
         waitForTopic(topic, 0);
     }
+    
+    default void createTopic(String topicName, int partitions, short replicas) {
+        try (Admin admin = createAdminClient()) {
+            admin.createTopics(Collections.singletonList(new NewTopic(topicName, partitions, replicas)));
```

Comment:
why not calling the wait?

Reply:
There is no specific reason.
Personally, I prefer user to waitTopic explicitly.
I am also fine to sync topic implicitly, just rename to CreateTopicWithSync.
WDYT?

Reply:
This is a "helper" method, so I feel a little sugar is good to users. We can add comments to this helper to remind users that this method is blocked until metadata get synced.

Reply:
Thanks for your explanation, updated.